src/components/NewsletterForm/NewsletterFormSubmit.tsx

```diff
@@ -18,7 +18,7 @@ export const NewsletterFormSubmit = ({
   <NewsletterFormItem type="submit">
     <Button
       className="newsletter-form__btn-submit"
-      disabled={disabled}
+      isDisabled={disabled}
```

Comment:
Noticed we need to change the prop name in the NewsletterFormSubmit component too - but don't worry about this, I think we will move this component to corporate-react soon.